src/questions/index.js

```diff
@@ -52,9 +54,13 @@ async function promptForCustomPorts(answers) {
     )
   }
 
-  if (isCakeshop(answers.cakeshop)) {
+  if (isCakeshop(answers.tools.includes('cakeshop'))) {
     answers.cakeshopPort = await getCustomizedCakeshopPort()
   }
+
+  if (isSplunk(answers.tools.includes('splunk'))) {
```

Comment:
doesn't look like the isSplunk function exists in NetworkConfig.js

Reply:
this was already resolved. you might be looking at an older commit.